### DISCUSSION TITLE
Incompatible mod set! How to fix it?

### DISCUSSION THREAD

**Discussion by vojta1292** [FabricMC/fabric-api#2539] [Player support]
I'm trying to download MiniHUD but it doesn't want to start and gives an error.

_Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Replace mod 'MaLiLib' (malilib) 0.13.0 with any 0.13.x version that is compatible with:
		 - minecraft 1.19
Unmet dependency listing:
	 - Mod 'MaLiLib' (malilib) 0.13.0 requires any version between 1.19.1 (inclusive) and 1.20 (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.19!
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:190)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:148)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)_

Where is the problem and how do I get the MiniHUD working?

**Comment by warjort** [comment]
As the error message says:
````
- Replace mod 'MaLiLib' (malilib) 0.13.0 with any 0.13.x version that is compatible with:
- minecraft 1.19
````

So you need the 1.19 version of malilib, i.e.  currently 1.19.0-0.13.0
https://www.curseforge.com/minecraft/mc-mods/malilib/files

If that doesn't fix it, you should contact the mod author.
They will know better how to resolve problems with their mod.

**Comment by unatried** [comment]
He has the right version as it says 0.13.0 but the 1.19 version on curseforge is also 0.13.0
https://www.curseforge.com/minecraft/mc-mods/malilib/files/all?filter-game-version=2020709689%3A9186

Do you have any other mods

**Reply by warjort** [reply]
There are multiple 0.13.0 versions prefixed with minecraft version.
The original poster had 1.19.1-0.13.0 instead of 1.19.0-0.13.0

**Reply by unatried** [reply]
found three other 0.13.0 version for 1.19 
https://www.curseforge.com/minecraft/mc-mods/malilib/files/3904734
https://www.curseforge.com/minecraft/mc-mods/malilib/files/3923052
https://www.curseforge.com/minecraft/mc-mods/malilib/files/3923052
